lang/csharp/src/apache/test/Reflect/TestLogMessage.cs

```diff
@@ -1,113 +0,0 @@
-/**
```

Comment:
Why did this get deleted?

Reply:
it was renamed to TestConverters - since it's doing more than testing the LogMessage model.